src/components/Header/DisclosureDialog.js

```diff
@@ -0,0 +1,74 @@
+import * as React from 'react';
+//import MaterialUI components
+import Button from '@mui/material/Button';
+import Dialog from '@mui/material/Dialog';
+import DialogActions from '@mui/material/DialogActions';
+import DialogContent from '@mui/material/DialogContent';
+import DialogContentText from '@mui/material/DialogContentText';
+import DialogTitle from '@mui/material/DialogTitle';
```

Comment:
Small suggestion, and really a preference around being more concise (just referencing the package folder)  vs. more explicit (referencing the package/component folder) . You could import these like this:
```
import {
  Button,
  Dialog,
  DialogActions,
  DialogContent,
  DialogContentText,
  DialogTitle,
} from '@mui/material'
```